sqlalchemy_ibmi/base.py

```diff
@@ -16,7 +16,103 @@
 # | Authors: Alex Pitigoi, Abhigyan Agrawal, Rahul Priyadarshi               |
 # | Contributors: Jaimy Azle, Mike Bayer                                     |
 # +--------------------------------------------------------------------------+
-"""Support for IBM Db2 database
+"""
+DBAPI Connection
+----------------
+This dialect uses the `pyODBC <https://github.com/mkleehammer/pyodbc>`_ DBAPI
+and the `IBM i Access ODBC Driver
+<https://www.ibm.com/support/pages/ibm-i-access-client-solutions>`_.
+
+Connect to Db2 for i using the following URL::
+
+    engine = create_engine("ibmi://username:pass@host/?connect_args")
+
+Connection arguments are passed following the URL query format. Please find
+more information on how to pass the arguments `here
+<https://en.wikipedia.org/wiki/Query_string>`_
```

Comment:
We could use similar text as https://docs.sqlalchemy.org/en/13/dialects/postgresql.html#dialect-postgresql-psycopg2-connect